improvement: simplify README.md on monorepo (#2848)
fern-api/fern: README.md

</div>

Fern is a platform for generating Stripe-level SDKs (client libraries) and Docs for your API. Get started by passing in your OpenAPI specification.

<div align="center">
    <img src="/fern/docs/images/overview.png" width="700" alt="Overview Diagram">
</div>

### 🌿 SDKs (API Clients)

> Our generators read in your OpenAPI spec as input and output SDKs. Fern handles publishing the generated SDKs to GitHub and package managers like NPM and PyPI.

Reviewer: ```suggestion
Our generators read in your OpenAPI spec to input and output SDKs. Fern handles publishing the generated SDKs to GitHub and package managers like NPM and PyPI.
```

Reviewer: Or _and_ perhaps instead of _as_? At least, _as_ doesn't read right to me. If you go with _and_, perhaps change _input and output_ to _input/output_ so that _and_ isn't repeated twice.

| fernapi/fern-java-sdk               | SDK generator for Java/Kotlin/JVM                              | 0.6.1          | -                                                            | -                                                                  |
| fernapi/fern-go-sdk                 | Go SDK generator                                               | 0.12.0         | -                                                            | -                                                                  |
| fernapi/fern-csharp-sdk             | C# SDK generator for .NET platforms                            | 0.0.0          | -                                                            | -                                                                  |
| fernapi/fern-ruby-sdk               | Ruby SDK generator                                             | 0.0.0          | [CHANGELOG.md](https://github.com/fern-api/fern/blob/main/generators/ruby/sdk/CHANGELOG.md) | -                                                                  |

For additional language support, visit [the Fern issues](https://github.com/fern-api/fern/issues) and let us know!

### 🌿 API Documentation

> Fern will build and host a documentation website with an auto-generated API reference. Write additional pages in markdown and have them versioned with git. Search, SEO, dark mode, and more are provided out-of-the-box.

Reviewer: ```suggestion
Fern can build and host a documentation website with an auto-generated API reference. Write additional pages in Markdown and version them with git. Search, SEO, dark mode, and more are provided out of the box.
```
